src/Licensing/ThroughputCollectorHostedService.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Particular.License
+{
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+    using Particular.License.Contracts;
+
+    class ThroughputCollectorHostedService : IHostedService
```

Comment:
It may make more sense to use a [BackgroudService](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/host/hosted-services?view=aspnetcore-8.0&tabs=visual-studio#backgroundservice-base-class), thoughts?

Reply:
Yeah haven't figured that part out yet - that's step 2 :-)